dev-lang/rapydscript-ng/rapydscript-ng-0.7.22.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+if [[ ${PV} == 9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/kovidgoyal/rapydscript-ng"
+	# this is nodejs software, so possibly unfixable. I am sorry.
+	# Instead, we will have to generate this with the live ebuild and upload assets
+	# using EGIT_OVERRIDE_COMMIT for the tagged version we need.
+	RESTRICT="network-sandbox"
```

Comment:
We can drop the RESTRICT here because `inherit git-r3` leads to Portage picking up `PROPERTIES="live"` from the eclass.

PROPERTIES="live" implies RESTRICT="network-sandbox". The problem that PROPERTIES="live" solves is this kind of case: we shouldn't assume "9999" => live, and we shouldn't assume live => 9999. So, it's a way of indicating in a non-VCS specific way that it doesn't solely use assets from SRC_URI and will try to access the network.